orangecontrib/text/datasets/url.tab

```diff
@@ -0,0 +1,20 @@
+url
+string
+meta
+http://www.dnaindia.com/india/report-activist-lawyer-indira-jaising-leads-protest-against-teesta-setalvad-s-victimisation-2061595

```

Comment:
Is this some special list of URLs? Should we rather include some more general links?